src/RadixDlt.NetworkGateway.PostgresIntegration/Models/ValidatorCumulativeEmissionHistory.cs

```diff
@@ -0,0 +1,94 @@
+ *
+ * Neither the licensor nor any current or former employees, officers, directors, partners,
+ * trustees, representatives, agents, advisors, contractors, or volunteers of the Licensor
+ * shall be liable for any direct or indirect, special, incidental, consequential or other
+ * losses of any kind, in tort, contract or otherwise (including but not limited to loss
+ * of revenue, income or profits, or loss of use or data, or loss of reputation, or loss
+ * of any economic or other opportunity of whatsoever nature or howsoever arising), arising
+ * out of or in connection with (without limitation of any use, misuse, of any ledger system
+ * or use made or its functionality or any performance or operation of any code or protocol
+ * caused by bugs or programming or logic errors or otherwise);
+ *
+ * A. any offer, purchase, holding, use, sale, exchange or transmission of any
+ * cryptographic keys, tokens or assets created, exchanged, stored or arising from any
+ * interaction with the Work;
+ *
+ * B. any failure in a transmission or loss of any token or assets keys or other digital
+ * artefacts due to errors in transmission;
+ *
+ * C. bugs, hacks, logic errors or faults in the Work or any communication;
+ *
+ * D. system software or apparatus including but not limited to losses caused by errors
+ * in holding or transmitting tokens by any third-party;
+ *
+ * E. breaches or failure of security including hacker attacks, loss or disclosure of
+ * password, loss of private key, unauthorised use or misuse of such passwords or keys;
+ *
+ * F. any losses including loss of anticipated savings or other benefits resulting from
+ * use of the Work or any changes to the Work (however implemented).
+ *
+ * You are solely responsible for; testing, validating and evaluation of all operation
+ * logic, functionality, security and appropriateness of using the Work for any commercial
+ * or non-commercial purpose and for any reproduction or redistribution by You of the
+ * Work. You assume all risks associated with Your use of the Work and the exercise of
+ * permissions under this License.
+ */
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace RadixDlt.NetworkGateway.PostgresIntegration.Models;
+
+[Table("validator_cumulative_emission_history")]
+internal class ValidatorCumulativeEmissionHistory
+{
+    [Key]
+    [Column("id")]
+    public long Id { get; set; }
+
+    [Column("from_state_version")]
+    public long FromStateVersion { get; set; }
+
+    [Column("validator_entity_id")]
+    public long ValidatorEntityId { get; set; }
+
+    [Column("epoch_number")]
+    public long EpochNumber { get; set; }
+
+    [Column("proposals_made")]
+    public long ProposalsMade { get; set; }
+
+    [Column("proposals_missed")]
+    public long ProposalsMissed { get; set; }
+
+    [Column("participation_in_active_set")]
+    public long ParticipationInActiveSet { get; set; }
```

Comment:
it's not clear what it holds, but i can't figure out better name. Maybe same as in our gateway response `EpochsActiveIn` or `EpochsAsActiveSetMember`

Reply:
I think it is very clear. It represents the cumulative sum (see table name) of all participation in active sets.

Reply:
wasn't for me, I had to analyze code to understand what it is. But if you insist we can leave it as it is.

Reply:
I'll leave it as it is.